packages/trader/src/AppV2/Components/TradeParameters/Multiplier/multiplier-wheel-picker.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useEffect } from 'react';
+import { ActionSheet, Text, WheelPicker, Skeleton } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { useTraderStore } from 'Stores/useTraderStores';
+import debounce from 'lodash.debounce';
+
+type TMultiplierWheelPickerProps = {
+    multiplier: ReturnType<typeof useTraderStore>['multiplier'];
+    multiplier_range_list: ReturnType<typeof useTraderStore>['multiplier_range_list'];
+    currency: ReturnType<typeof useTraderStore>['currency'];
+    commission: ReturnType<typeof useTraderStore>['commission'];
+    setMultiplier: (multiplier: number) => void;
+};
+
+const debouncedSetMultiplier = debounce((setMultiplier, multiplier) => {
+    setMultiplier(multiplier);
+}, 200);
+
+const MultiplierWheelPicker = ({
+    multiplier,
+    multiplier_range_list = [],
+    currency,
+    commission,
+    setMultiplier,
+}: TMultiplierWheelPickerProps) => {
+    const multiplier_array = multiplier_range_list.map(item => ({ value: item.text }));
+    const initial_multiplier = React.useRef<number>();
+    const selected_multiplier = React.useRef<number>(multiplier);
+
+    useEffect(() => {
+        if (!initial_multiplier.current && multiplier) {
+            initial_multiplier.current = multiplier;
+        }
```

Comment:
```suggestion
    const initial_multiplier = React.useRef<number>(multiplier);
    const selected_multiplier = React.useRef<number>(multiplier);

    useEffect(() => {
```

Can we set the multiplier as the default value in the declaration phase?